command-expansion-server/test/sequence-generation.spec.ts

```diff
@@ -546,3 +546,61 @@ describe('sequence generation', () => {
   }, 10000);
 
 });
+
```

Comment:
Does this test factor in the serialization changes implicitly? In that case, why didn't the pre-existing tests for this functionality suffice?

Reply:
We did not have a test that utilized the temporal objects from the simulated activity, just Temporal objects that were created inline.

The two lines below show in the expansion where we actually exercise the serialization/deserialization of Temporal objects on the simulated activity

https://github.com/NASA-AMMOS/aerie/pull/226/files/810fba7da16c4cd8db39f5d07200fb506259d2c3..a90d35c0167dffd5f918bc00bc99d0d77abc0e7d#diff-8e57f67494408300790faaeea0a418e551a6d8a64425748d1452c6ff5dad82e9R561-R562